kitsune/products/views.py

```diff
@@ -96,6 +97,9 @@ def document_listing(request, product_slug, topic_slug, subtopic_slug=None):
     thirty_days_ago = datetime.now() - timedelta(days=30)
     for document in documents:
         document["is_past_thirty_days"] = document["created"] < thirty_days_ago
+        document["is_first_revision"] = (
+            Revision.objects.filter(document=document["id"]).count() == 1
```

Comment:
I was testing the PR and realized that simply creating a new revision that hasn't been approved yet changed `Created` to `Last updated`, which seems wrong. So I'm thinking this should change slightly to count the number of approved revisions.

Reply:
Great catch. Updated.